tests/integration/test_grpc_protocol_ssl/test.py

```diff
@@ -42,6 +42,9 @@
         "configs/server-cert.pem",
         "configs/ca-cert.pem",
     ],
+    # Bug in TSAN reproduces in this test https://github.com/grpc/grpc/issues/29550#issuecomment-1188085387
+    # second_deadlock_stack -- just ordinary option we use everywhere, don't want to overwrite it
+    env_variables={"TSAN_OPTIONS": "report_atomic_races=0 second_deadlock_stack=1"},
```

Comment:
IMHO it looks clearer when a single PR doesn't contain changes unrelated to each other. I mean you're adding this TSAN option and at the same time you're randomizing the tests.

Reply:
I do not randomize tests. I just add package that helps to run tests in random order without `pip install` every time. I just did not want to spend CI resources for noop change.